modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.module

```diff
@@ -94,6 +96,24 @@ function social_profile_privacy_form_social_profile_admin_settings_form_alter(&$
     '#format' => $config->get('disclaimer.format'),
   ];
 
+  // Add warning to inform about update all profiles after submit form.
```

Comment:
/s

`// Add warning to inform about update all profiles after submit form.`

to 

`// Add warning for site_manager to inform about updating all the profiles after form is submitted.`

Reply:
Updated.